src/Microsoft.DotNet.Tools.Repl.Csi/Program.cs

```diff
@@ -150,24 +148,21 @@ private static int Run(string script, string targetFramework, string buildConfig
                     }
 
                     string responseFile = CreateResponseFile(projectContext, buildConfiguration, tempOutputDir);
-                    csiArgs.Append($"@\"{responseFile}\" ");
+                    csiArgs.Add($"@\"{responseFile}\" ");
                 }
 
                 if (string.IsNullOrEmpty(script) && !remainingArguments.Any())
                 {
-                    csiArgs.Append("-i");
+                    csiArgs.Add("-i");
                 }
                 else
                 {
-                    csiArgs.Append(script);
+                    csiArgs.Add(script);
                 }
 
-                foreach (string remainingArgument in remainingArguments)
-                {
-                    csiArgs.Append($" {remainingArgument}");
-                }
+                csiArgs.AddRange(remainingArguments);
 
-                return Command.Create(csiExe, csiArgs.ToString())
+                return Command.Create("csi", csiArgs)
```

Comment:
No, `Command.Create` will resolve the location of `csi`. It will first attempt to `ResolveExecutablePathFromProject` and then `ResolveExecutableFromPath`. This makes the location of `csi` resolve in the same manner as the other `dotnet-*` commands are resolved.